collaboration-service/server.ts

```diff
@@ -0,0 +1,55 @@
+import { Server } from "socket.io";
+import mongoose from "mongoose";
+import { DocumentModel, DocumentType } from './src/models/document'
+import { COLLABORATION_SERVICE_PORT, COLLABORATION_SERVICE_MONGODB_URI } from './config'
+
+const FRONTEND_PORT = 5173
```

Comment:
i think its bttr to indicate frontend port in the env file because it might be different from frontend

Reply:
oki done